Exercise_B/index.html

```diff
@@ -7,21 +7,62 @@
     <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
 
     <!-- Link and CDN required for bootstrap -->
-
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
     <!-- Google Font - Lato -->
     <link
       href="https://fonts.googleapis.com/css?family=Lato"
       rel="stylesheet"
     />
     <!-- Custom Stylesheet -->
+    <!-- CSS only -->
+    
     <link rel="stylesheet" href="./assets/style/stylesheet.css" />
+    <!-- JavaScript Bundle with Popper -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
   </head>
 
   <body>
     <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
+    <div class="container1">
+
+      <div class ="row" style="height: 50px">

```

Comment:
Hello Anson, 
For this i would suggest giving it a width of 100vw so that it fits the whole width of any screen